script.js

```diff
@@ -64,7 +64,7 @@ function findElement(e) {
 
 function updateCanvas() {
   let paths = calculateFieldLines();
-  document.getElementById('lines').getContext('2d').clearRect(0, 0, c.width, c.height);
+  document.getElementById('lines').getContext('2d').clearRect(0, 0, document.getElementById('lines').width, document.getElementById('lines').height);
```

Comment:
It would save typing (or cut and pasting) and make this a bit more readable to use a variable, e.g.

``` js
let lines = document.getElementById('lines');
lines.getContext('2d').clearRect(0, 0, lines.width, lines.height);
```